Formula/m/mise.rb

```diff
@@ -42,7 +42,7 @@ def install
 
     system "cargo", "install", *std_cargo_args
     man1.install "man/man1/mise.1"
-    generate_completions_from_executable(bin/"mise", "completion")
+    generate_completions_from_executable(bin/"mise", "completion", "--include-bash-completion-lib")
```

Comment:
I think this will break for zsh and fish, only required for bash right?

Reply:
I've just tested this and it only adds the extra libs if bash is specified.

So no, it doesn't break for other shells.

Reply:
Isn't this something that belongs in the user's .bash.rc? Looks like it's optional so the same library isn't sourced over-and-over at shell initialization. Maybe some clarification on what @jdx's (upstream) intention was with this flag would be helpful?

Additionally, does this work for both bash 3 and 5?

Reply:
I thought homebrew had this as a separate package. How do other CLI's include it? I wouldn't think you'd want my vendored version

Reply:
It appears there are two bash-completion recipes, both sourced from https://github.com/scop/bash-completion

- bash-completion (v1.3, for bash versions < 4.2)
- bash-completion@2 (v2.15.0 for bash versions 4.2+)

I imagine it may be difficult to optionally install one of these packages when mise is installed?